packages/commons/src/events/FieldConfig.ts

```diff
@@ -60,13 +60,15 @@ const BaseField = z.object({
     .optional(),
   dependsOn: z.array(FieldId).default([]).optional(),
   label: TranslationConfig,
-  hideLabel: z.boolean().default(false).optional()
+  hideLabel: z.boolean().default(false).optional(),
+  hideOnReview: z.boolean().default(false).optional()
```

Comment:
Can we make these so the Input component in registered-fields/ controls if the field should be shown? FileOutput I think has something like this already: https://github.com/opencrvs/opencrvs-core/blob/68805dd6d9742aaca4be41340a83b207798d3fd3/packages/client/src/v2-events/components/forms/inputs/FileInput/FileInput.tsx#L75